javav2/example_code/timestream/src/main/java/com/timestream/write/CreateDatabase.java

```diff
@@ -0,0 +1,73 @@
+//snippet-sourcedescription:[CreateDatabase.java demonstrates how to create a database.]
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Timestream]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[05-04-2022]
+//snippet-sourceauthor:[scmacdon- AWS]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.timestream.write;
+
+//snippet-start:[timestream.java2.create_db.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.timestreamwrite.TimestreamWriteClient;
+import software.amazon.awssdk.services.timestreamwrite.model.CreateDatabaseRequest;
+import software.amazon.awssdk.services.timestreamwrite.model.TimestreamWriteException;
+//snippet-end:[timestream.java2.create_db.import]
+
+/**
+ * To run this Java V2 code example, ensure that you have set up your development environment, including your credentials.
+ *
+ * For information, see this documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+
+public class CreateDatabase {
+
+    public static void main(String[] args) {
+
+        final String usage = "\n" +
+                "Usage: " +
+                "   <dbName> \n\n" +
+                "Where:\n" +
+                "   dbName - the name of the database.\n\n" ;
```

Comment:
Capitalize leading "The" 

the name --> The name

Reply:
fixed

Reply:
2x checked